contracts/minter/src/contract.rs

```diff
@@ -191,6 +191,10 @@ pub fn execute_set_whitelist(
         return Err(ContractError::AlreadyStarted {});
     }
 
+    if config.whitelist.is_some() {
+        return Err(ContractError::WhitelistAlreadySet {});
```

Comment:
So the intention here is to only let the whitelist be set once? No way to update it once it's been set?